LinqLesson/Program.cs

```diff
@@ -0,0 +1,192 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using static System.Console;
+
+namespace LinqLesson
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			var persons = JsonConvert.DeserializeObject<IEnumerable<Person>>(File.ReadAllText("data.json"));
+
+            var allTagsSelected = persons.Select(x => x.Tags);
+            var allTags = persons.SelectMany(x => x.Tags);
+            var tagsGroups = allTags.GroupBy(x => x);
+            var mainTag = tagsGroups.MaxBy(x => x.Count()).Key;
+
+            var personsWithSameTag = persons.Where(x => x.Tags.Contains(mainTag)).Count();
+            WriteLine($"Persons with same tag '{mainTag}' are {personsWithSameTag}");
+
+            //Part1
+            var MaxLatitudePerson = persons.MaxBy(x => x.Latitude);

```

Comment:
code convention violated
```suggestion
            var maxLatitudePerson = persons.MaxBy(x => x.Latitude);

```